homeassistant/components/stookwijzer/config_flow.py

```diff
@@ -0,0 +1,48 @@
+"""Config flow to configure the Stookwijzer integration."""
+from __future__ import annotations
+
+from typing import Any
+
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow
+from homeassistant.const import CONF_LATITUDE, CONF_LONGITUDE, CONF_NAME
+from homeassistant.data_entry_flow import FlowResult
+import homeassistant.helpers.config_validation as cv
+
+from .const import DOMAIN
+
+
+class StookwijzerFlowHandler(ConfigFlow, domain=DOMAIN):
+    """Config flow for Stookwijzer."""
+
+    VERSION = 1
+
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """Handle a flow initialized by the user."""
+
+        if user_input is not None:
+            await self.async_set_unique_id(
+                f"{user_input[CONF_LATITUDE]}-{user_input[CONF_LONGITUDE]}"
+            )
+            self._abort_if_unique_id_configured()
```

Comment:
This is not a good unique ID in general, a small change and it will make it different (which is just as affective as not doing it at all).

```suggestion
```